packet/flatcar-linux/kubernetes/bootkube.tf

```diff
@@ -22,4 +22,14 @@ module "bootkube" {
   enable_aggregation    = "${var.enable_aggregation}"
 
   certs_validity_period_hours = "${var.certs_validity_period_hours}"
+
+  container_images = {
+    calico           = "calico/node:v3.9.2"
```

Comment:
Oh, current version should be `calico/*:v3.10.1`

Reply:
Actually, I'm talking about all images here. Will we have to always sync it up now when we update bootkube module?

Reply:
We can also change the bootkube defaults to the values here.

Reply:
Except for the `os_arch` suffix, this would need some more changes for bootkube.

Reply:
Yeah, I think it would be better to propagate that to, I assume `terraform-render-bootkube` module, so we have single source of truth across our code.

Reply:
Have changed it and will test now.

Reply:
Works